app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -377,6 +390,12 @@ class SearchFragment : Fragment(), UserInteractionHandler {
         }
     }
 
+    private fun dismissAndResetFocus() {
+        view?.search_scan_button?.isChecked = false
```

Comment:
Question: what are the scenarios when this would be called and view would be null? Just wanted to make sure I understand what's going on here.

Reply:
Great catch! This was a copy/paste from `onBackPressed` above. They both should be `toolbarView.view.search_scan_button` or just `search_scan_button`